### ISSUE TITLE
不支持largeTitle

### ISSUE THREAD

**Issue by nyssance** [CRAnimation/CRRefresh#30]
不过至今也没发现别的支持largeTitle的刷新和上推隐藏导航栏

**Comment by jstngoulet** [comment]
Not sure if this is the same issue due to translation, but, I am getting an issue where the animator does not display when there is large navigation title. In addition, Ramotion animator is not working on 1.1.2. I am just getting a tableview frame size of my ball color.

**Comment by Ascendance** [comment]
Yes, translation says could not find refresher that supports large titles today thus far. 

I have ideas but could not find a way to _stop_ the entire title from bouncing down. 

Maybe invent a customizable large title plugin that mimics apple's prefer large tittles? 😂
